scripts/release/list_backports.py

```diff
@@ -0,0 +1,340 @@
+
+    setattr(args, 'token', token)
+
+    return args
+
+
+class Backport(object):
+    def __init__(self, repo, base, pulls):
+        self._base = base
+        self._repo = repo
+        self._issues = []
+        self._pulls = pulls
+
+        self._pulls_without_an_issue = []
+        self._pulls_with_invalid_issues = {}
+
+    @staticmethod
+    def by_date_range(repo, base, start_date, end_date, excludes):
+        """Create a Backport object with the provided repo,
+        base, start datetime object, and end datetime objects, and
+        list of excluded PRs"""
+
+        pulls = []
+
+        unfiltered_pulls = repo.get_pulls(
+            base=base, state='closed')
+        for p in unfiltered_pulls:
+            if not p.merged:
+                # only consider merged backports
+                continue
+
+            if p.closed_at < start_date or p.closed_at >= end_date + timedelta(1):
+                # only concerned with PRs within time window
+                continue
+
+            if p.number in excludes:
+                # skip PRs that have been explicitly excluded
+                continue
+
+            pulls.append(p)
+
+        # paginated_list.sort() does not exist
+        pulls = sorted(pulls, key=lambda x: x.number)
+
+        return Backport(repo, base, pulls)
+
+    @staticmethod
+    def by_included_prs(repo, base, includes):
+        """Create a Backport object with the provided repo,
+        base, and list of included PRs"""
+
+        pulls = []
+
+        for i in includes:
+            try:
+                p = repo.get_pull(i)
+            except Exception:
+                p = None
+
+            if not p:
+                logging.error(f'{i} is not a valid pull request')
+                return None
+
+            if p.base.ref != base:
+                logging.error(
+                    f'{i} is not a valid pull request for base {base} ({p.base.label})')
+                return None
+
+            pulls.append(p)
+
+        # paginated_list.sort() does not exist
+        pulls = sorted(pulls, key=lambda x: x.number)
+
+        return Backport(repo, base, pulls)
+
+    @staticmethod
+    def sanitize_title(title):
+        # TODO: sanitize titles such that they are suitable for both JSON and ReStructured Text
+        # could also automatically fix titles like "Automated backport of PR #1234"
+        return title
+
+    def print(self):
+        for i in self.get_issues():
+            title = Backport.sanitize_title(i.title)
+            # * :github:`38972` - logging: Cleaning references to tracing in logging
+            print(f'* :github:`{i.number}` - {title}')
+
+    def print_json(self):
+        issue_objects = []
+        for i in self.get_issues():
+            obj = {}
+            obj['id'] = i.number
+            obj['title'] = Backport.sanitize_title(i.title)
+            obj['url'] = f'https://github.com/zephyrproject-rtos/zephyr/pull/{i.number}'
```

Comment:
need to use the org and repo names from the arg.

Reply:
Fixed!